src/pages/SmartWeaveExplorer/Contract/Contract.vue

```diff
@@ -149,6 +149,10 @@
             </div>
           </div>
         </div>
+        <div class="mt-6">
+          <p class="json-header">Contract tags:</p>
+          <json-viewer :value="tags" :expand-depth="2" copyable sort theme="json-theme"></json-viewer>
+        </div>
```

Comment:
I guess the best solution here would be to make a seaparate tab (next to Transactions/Code/State) and place it the json over there.

Reply:
Done, I've added ContractTags component, which is now new tab, next to code,state..